docs/wiki/Solution creation.md

```diff
@@ -35,6 +36,77 @@ There are several open-source repositories that leverage the CARML library today
 | [AKS Baseline Automation](https://github.com/Azure/aks-baseline-automation) | Repository for the AKS Landing Zone Accelerator program's Automation reference implementation |
 | [DevOps Self-Hosted](https://github.com/Azure/DevOps-Self-Hosted) | - Create & maintain images with a pipeline using the Azure Image Builder service <p> - Deploy & maintain Azure DevOps Self-Hosted agent pools with a pipeline using Virtual Machine Scale Set|
 
+# General solution creation
```

Comment:
I'd suggest to move the added guidelines after the orchestration section, since we're also referencing it

Reply:
done